src/module/actor/actor.js

```diff
@@ -268,14 +268,7 @@ export class ActorSFRPG extends Mix(Actor).with(ActorConditionsMixin, ActorCrewM
      */
     async _onDelete(options, userId) {
         for (const item of this.items) {
-            if (item.type === "effect") {
-                const effect = game.sfrpg.timedEffects.get(item.uuid);
-                if (!effect) continue;
-
-                // Need to pass the item since the item has already been deleted from the server
-                effect.delete(item);
-            }
-
+            game.sfrpg.timedEffects.get(item.uuid)?.delete();
```

Comment:
Does not passing the `item` work fine?

Reply:
Like if `item` is `undefined` or `null`? I suppose not because of the direct access of `item.uuid`. Or do you mean if the UUID comes back `undefined`, `null` or empty? `Map.get` can handle that just fine and will return `undefined`. The new code should behave identically to the old code, just without the check of the item's type.

Reply:
oh. Sorry. I obviously misunderstood. Yeah, there's no harm in not passing `item` to `delete()`. It looks like that comment was referring to something old, because `delete()` already took zero parameters when I showed up. I didn't change that in this PR.